### ISSUE TITLE
Gitlab лучше для менеджмента

### ISSUE THREAD

**Issue by kroq86** [philippranzhin/peredelanoconf#6]
Можно назначить исполнителя, выставить ему estimation 2 дня например. Много досок в одном проекте - 1 для оргов, 1 для менеджеров и тд. Есть API по которому например можно запрещать завершение задачи без аппрува мердж реквеста. Есть майлостоуны куда например запихать все задачи до сентября. Есть лейблы для приоритезации задач или для поиска, к примеру дизайнер по поиску нажал на лейбл дизайн и увидел все задачи которые он может взять с лейблом высокого приоритета. ну вы в общем советую.

**Comment by proggga** [comment]
Согласен, лучше пойти в гитлаб клауд либо поставить свой, гитлаб золото

**Comment by nixel2007** [comment]
В гитхабе тоже есть несколько досок, автоматизация, полиси на мерж реквесты, майлстоуны и проч.

**Comment by philippranzhin** [comment]
Мы гх взяли не из-за удобство для менеджмента, в первую очередь, нам важно аудиторию увеличивать, и в этом смысле гх далеко впереди
